main/openvswitch/ovs-modules.initd

```diff
@@ -3,23 +3,18 @@
 MODULES="openvswitch"
 
 depend() {
-        before ovsdb-server
+	before ovsdb-server
 }
 
-
 start() {
-        ebegin "Loading Openvswitch kernel modules"
-        for mod in $MODULES; do
+	ebegin "Loading Openvswitch kernel modules"
+	for mod in $MODULES; do
 		modprobe -q $mod
 	done
 	eend $?
 }
 
 stop() {
-        ebegin "Unloading Openvswitch kernel modules"
-        for mod in $MODULES; do
-		rmmod $mod
-	done
-        eend $?
+	# modules cannot be removed with ovs running
```

Comment:
Please write what you stated in PR description here, i.e. cannot be removed without restart. That’s important difference.

Reply:
changed